stan/math/prim/fun/add.hpp

```diff
@@ -8,6 +8,21 @@
 namespace stan {
 namespace math {
 
+/**
+ * Return the sum of the specified scalars.
+ *
+ * @tparam ScalarA type of the first scalar
+ * @tparam ScalarB type of the second scalar
+ * @param a first scalar
+ * @param b second scalar
+ * @return the sum of the scalars
+ */
+template <typename ScalarA, typename ScalarB,
```

Comment:
Thanks for revising.

One more thing I noticed.  These should be `const A& a` and `const B& b` because `a` and `b` could be autodiff variables.   Otherwise, the constraint should be that they're arithmetic.